packages/react/src/search/search.react.tsx

```diff
@@ -0,0 +1,20 @@
+import { c, classy } from '@onfido/castor';
+import { Icon } from '@onfido/castor-react';
+import React, { forwardRef } from 'react';
+import { Input, InputProps } from '../input/input.react';
+
+export const Search = forwardRef(_Search) as typeof _Search;
```

Comment:
What do you think if this is added:

```suggestion
/**
 * Please note that this component uses an Icon component that requires an SVG
 * sprite to be inlined in your app.
 * 
 * More details: https://github.com/onfido/castor-icons#use-with-plain-code
 */
export const Search = forwardRef(_Search) as typeof _Search;
```